platform/mime/core/platform-mime-core-impl/src/main/java/ddf/mime/mapper/MimeTypeMapperImpl.java

```diff
@@ -66,8 +71,7 @@ public int compare(MimeTypeResolver o1, MimeTypeResolver o2) {
     /**
      * Constructs the MimeTypeMapper with a list of {@link MimeTypeResolver}s.
      *
-     * @param mimeTypeResolvers
-     *            the {@link List} of {@link MimeTypeResolver}s
+     * @param mimeTypeResolvers the {@link List} of {@link MimeTypeResolver}s
```

Comment:
I don't really see how this is a TODO. It's more of an explanation...I don't think I have a way of asking the original author so I think I'm just going to remove the TODO tag...